include/osg/ref_ptr

```diff
@@ -22,6 +22,10 @@
 #include <string>
 #endif
 
+#if __cplusplus >= 201103L
+#include <utility>
```

Comment:
This header shouldn't be needed, `std::move` in C++11 isn't in `utility`

Reply:
If it's not [utility](https://en.cppreference.com/w/cpp/utility/move) then which one? It's a part of the standard library so some header should contain it.

Reply:
All the sources I see say it's in <utility>. If it's in a different header on your machine, that's probably not something safe to rely on elsewhere.

Reply:
There is no header you need to include for this `std::move`, it is always available.
The one in utility is a different function from moving all elements of one container to another.

Reply:
Nevermind, I confused this with `std::move` from `<algorithm>`

Reply:
Do you have an example of such compiler? `std::move` is a function not operator. And compiler [can't](https://godbolt.org/z/67x3fsPzs) get it out of nowhere including `std` namespace. For example libstdc++ that provides standard library for gcc defines it [explicitly](https://github.com/gcc-mirror/gcc/blob/406f58e1e38e92e4b881f3666b596843da308783/libstdc%2B%2B-v3/include/bits/move.h#L101-L105). libc++ does it different but when running only the compiler preprocessor it's there:
```c++
template <>
struct is_destructible<void>
    : public std::__1::false_type {};
# 2564 "/home/elsid/dev/llvm-project/build/clang/release/install/include/c++/v1/type_traits" 3
template <class _Tp>
inline __attribute__ ((__visibility__("hidden"))) __attribute__ ((__exclude_from_explicit_instantiation__)) constexpr
typename remove_reference<_Tp>::type&&
move(_Tp&& __t) noexcept
{
    typedef __attribute__((nodebug)) typename remove_reference<_Tp>::type _Up;
    return static_cast<_Up&&>(__t);
}
```

Without including utility gcc and clang give error:
```
error: use of undeclared identifier 'std'
    A b = std::move(b);
          ^
```

There is another [move](https://en.cppreference.com/w/cpp/algorithm/move) that is similar to `std::copy` except it moves elements instead of copying but it's defined in the algrorithm header.

Reply:
I've merged this submission and am testing, but I can't see any use of std::move in the ref_ptr header, so why does it need to include a header for a function that it doesn't use?

Seems like there is no need for the include at all.  If code elsewhere relies upon std::move() then it should be the one to include<utility> not include/osg/ref_ptr.